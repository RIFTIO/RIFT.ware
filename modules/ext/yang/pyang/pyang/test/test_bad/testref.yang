module testref {
  namespace "http://example.com/testref";
  prefix tr;

  leaf foo {
    type leafref {
      path "/foo";
    }
  }
}
