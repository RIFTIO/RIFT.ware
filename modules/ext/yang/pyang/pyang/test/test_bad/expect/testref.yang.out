testref.yang:7: error: CIRCULAR_DEPENDENCY
